doc/source/reference/series.rst

```diff
@@ -110,7 +110,7 @@ Binary operator functions
    Series.product
    Series.dot
 
-Function application, groupby & window
+Function application, GroupBy & window
```

Comment:
What's the reason for this one? Just my opinion, but I think this one was correct, and we don't want to capitalize `groupy` to `GroupBy`.

Reply:
I might have wrongly seen it with capital letters or something, re-updating PR

Reply:
I see your point. to me it feels that there is the class `GroupBy`, but here it's not talking about the class, but about the concept. I guess we can use `group by` or `groupping by` to disambiguate. But not that important, we can merge it like this and change it later if we think that makes sense.